src/formpack/schema/fields.py

```diff
@@ -424,6 +458,72 @@ def get_substats(
 
 
 class TextField(ExtendedFormField):
+    @property
+    def _is_transcript(self):
+        return getattr(self, 'analysis_type', '') == ANALYSIS_TYPE_TRANSCRIPT
+
+    @property
+    def _is_translation(self):
+        return getattr(self, 'analysis_type', '') == ANALYSIS_TYPE_TRANSLATION
+
+    def get_labels(
```

Comment:
Did we say we would put "private/protected" method after public ones? 
We should keep the methods sorted alphabetically too. This file starts to be really big, It's easier to find method inside classes.

Reply:
Good point 👍